server/util/live-loan/live-loan-fetch.js

```diff
@@ -294,6 +304,11 @@ async function parseFilterStringLegacy(filterString) {
 					if (theme) {
 						addArrayFilterValue(name, theme.name);
 					}
+				} else if (name === 'tag') {
+					const tag = findFilterOption(tags, name, value);
+					if (tag) {
+						addArrayFilterValue(name, tag.name);
```

Comment:
The legacy search expects an array of tag IDs, using the filter name 'loanTags', so this line should be changed to `addArrayFilterValue('loanTags', tag.id);`. This will mean the tests need to be updated as well, for example: `await testFilterParsing('tag_u.s. black-owned businesses', { loanTags: [43] });`